apps/editors/templates/editors/emails/unlisted_to_sandbox.ltxt

```diff
@@ -0,0 +1,12 @@
+{% extends "editors/emails/base.ltxt" %}{% block content %}
+Your add-on, {{ name }} {{ number }}, didn't pass review and therefore won't be signed.
```

Comment:
Are there next steps for the developer that we should include? This seems a bit abrupt.